actions/views/root.js

```diff
@@ -18,3 +21,17 @@ export function loadMeAndConfig() {
         return Promise.all(promises);
     };
 }
+
+export function loadTranslations(locale, url) {
+    return (dispatch) => {
+        Client4.getTranslations(url).then((translations) => {
+            dispatch({
+                type: ActionTypes.RECEIVED_TRANSLATIONS,
+                data: {
+                    locale,
+                    translations,
+                },
+            });
+        }).catch(() => {}); // eslint-disable-line no-empty-function
```

Comment:
I just followed the original code for this, so I didn't add anything new here. The only errors that could be thrown would be due to a network error.

This action still isn't great because it does go through `Client4`, but it doesn't use any of the features of `Client4` since it's just passing a complete URL in. It's basically a glorified fetch.